File: app/src/main/java/com/example/openroom/activity/LogRegActivity.java
package com.example.openroom.activity;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;

import android.os.Bundle;
import android.view.WindowManager;

import com.example.openroom.R;
import com.example.openroom.fragment.AuthFragment;

public class LogRegActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_log_reg);
        AuthFragment authFragment = new AuthFragment();
        FragmentManager fragmentManager = getSupportFragmentManager();
        fragmentManager.beginTransaction()
                .add(R.id.fr_place, authFragment)
                .commit();
        setStatusBarTranslucent(true);
    }
    protected void setStatusBarTranslucent(boolean makeTranslucent) {
        if (makeTranslucent)
        {
            getWindow().addFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
        } else {
            getWindow().clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
        }
    }
}

File: app/src/main/java/com/example/openroom/activity/LogRegActivity.kt
package com.example.openroom.activity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.openroom.R
import com.example.openroom.fragment.AuthFragment
import android.view.WindowManager

class LogRegActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_log_reg)
        setStatusBarTranslucent(true)
    }

    protected fun setStatusBarTranslucent(makeTranslucent: Boolean) {
        if (makeTranslucent) {
            window.addFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS)
        } else {
            window.clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS)
        }
    }
}